README.md

```diff
@@ -72,7 +72,6 @@ create a new Maven project with this simple pom:
             <version>1.2.3</version>
             <type>jar_or_xar</type>
         </dependency>
-        ...
```

Comment:
What do you think about making the dots a comment so that they are syntactically valid but still convey the original meaning?

Reply:
For what reason? In meantime the above comment <!-- More XWiki Extension you like to have -->  is mentioned that user can add one or more extensions. And the dots come after that, so user can miss them by accident and only see them during compile as error. For blog extension there's separate comment.

Reply:
@petrenkonikita112263 are you sure you need this tool ? People generally use the [XIP packaging](https://dev.xwiki.org/xwiki/bin/view/Community/ExtensionPlugin/#HXIPMojoandextension) these days for offline install needs.

Reply:
I'd tested it last week as another alternative option for offline installation.